consensus/istanbul/validator.go

```diff
@@ -114,8 +87,10 @@ type ValidatorSet interface {
 	// Get the minimum quorum size
 	MinQuorumSize() int
 
-	// Return the validator array
+	// List returns all the validators
 	List() []Validator
+	// ListAddresses returns all the validators addresses
+	ListAddresses() []common.Address
```

Comment:
It may be better to avoid adding this function, as the result can be obtained by calling `MapValidatorsToAddresses(valSet.List())` and this interface is already bloated.

Reply:
done.

I did this since `MapValidatorsToAddresses` seems "hard to find"... it's on the istanbul package, imo all validator related things should live in `validator` pkg.

Reply:
I think the desired to not add more functions to this interface outweighs that though. Especially because this is fairly simple function and in general Go code prefers to include the loop inline instead of creating these utility function (which I think shows a weakness of the language, but generally aggree with given the tools Go provides)